### PR TITLE
can add friends and see all of my friends in the dashboard

### PR DESCRIPTION
**Pull request by pramish**

### CODE REVIEW

File: client/src/components/dashboard/add-friends/add-friends.jsx [hunk 1]
```diff
@@ -23,10 +23,13 @@ export const AddFriendsForm = (props) => {
 			{
 			name
 			email
+      date_of_birth
+      phone_number
```
Comment (boltdynamics) [inline client/src/components/dashboard/add-friends/add-friends.jsx:27]: Consistent spacing please

File: client/src/components/dashboard/add-friends/add-friends.jsx [hunk 2]
```diff
@@ -36,7 +39,7 @@ export const AddFriendsForm = (props) => {
     })
       .then((res) => res.json())
       .then((data) => {
-        console.log("What is the value of add???? ", add);
+        console.log("What is the value of add???? ", data);
```
Comment (boltdynamics) [inline client/src/components/dashboard/add-friends/add-friends.jsx:42]: Did you want to change the description too?

File: rootValue/rootValue.js [hunk 3]
```diff
@@ -34,8 +34,8 @@ module.exports = RootValue = {
   sendEmail: async (args, req) => {
     try {
       const isAuth = await req.isAuth;
-      if (!isAuth) throw new Error('Please autheticate');
-      console.log('Yaha aako xa ra??', req.userId);
+      if (!isAuth) throw new Error("Please autheticate");
```
Comment (boltdynamics) [inline rootValue/rootValue.js]:
```suggestion
      if (!isAuth) throw new Error("Please authenticate");
```

File: rootValue/rootValue.js [hunk 4]
```diff
@@ -34,8 +34,8 @@ module.exports = RootValue = {
...
+      console.log("Yaha aako xa ra??", req.userId);
```
Comment (boltdynamics) [inline rootValue/rootValue.js]: Is the user/developer always going to be Nepali?

File: server.js [hunk 5]
```diff
@@ -1,20 +1,22 @@
...
 const isAuth = require("./auth/auth");
 const getDateof = require("./helper/send_message_helper");
 
+app.delete("/hello", (req, res) => {
+  console.log("I am hit.");
+});
+
 app.use((req, res, next) => {
-  res.setHeader("Access-Control-Allow-Origin", "*");
+  res.setHeader("Access-Control-Allow-Origin", "http://localhost:3000");
```
Comment (boltdynamics) [inline server.js]: Does this mean your server is always going to run from your local machine?

File: client/src/components/dashboard/add-friends/add-friends.jsx (hunk 2 above)
Comment (pramish) [reply client/src/components/dashboard/add-friends/add-friends.jsx:42]: This has been deleted.

File: rootValue/rootValue.js (hunk 3 above)
Comment (pramish) [reply rootValue/rootValue.js]: Comments has been deleted.

File: rootValue/rootValue.js (hunk 4 above)
Comment (pramish) [reply rootValue/rootValue.js]: Comments has been deleted.

File: server.js (hunk 5 above)
Comment (pramish) [reply server.js]: Have updated the client url.

**Comment by pramish** [comment]
I have updated the pr.

**Review by boltdynamics — APPROVED** [review]
Legend, looks good 🚀

**Pull request merged**